src/modes/simple_select.js

```diff
@@ -168,6 +172,7 @@ module.exports = function(ctx, startingSelectedIds) {
       });
 
       this.on('mouseup', () => true, function(e) {
+        if (dragging) emitModify();
```

Comment:
`dragging` doesn't mean we've dragged, just that we can drag. We might want to rename this to `canDrag`.